app/src/main/res/layout-land/profile_reset_pin_activity.xml

```diff
@@ -86,6 +87,7 @@
             android:layout_marginStart="168dp"
             android:layout_marginTop="32dp"
             android:layout_marginEnd="168dp"
+            android:textDirection="locale"
```

Comment:
This not required ProfileInputView already supports RTL

Reply:
@ShridharGoel Please reply to these comments.

Reply:
This has been updated.